pubobot/bot.py

```diff
@@ -1361,16 +1275,17 @@ def lastgame(self, member, args):
         else:
             l = self.lastgame_cache
         if l:
-            n = l[0]
+            pickup_num = l[0]
             ago = datetime.timedelta(seconds=int(time.time() - int(l[1])))
-            gt = l[2]
+            gametype = l[2]
+
+            ## Can't use memberformatter here. players, alpha_players, beta_players are strings rather than arrays
             if l[4] and l[5]:
-                players = "[{0}] vs [{1}]".format(l[4], l[5])
+                players = "{0}\n{1}".format(l[4], l[5])
             else:
-                players = ", ".join(l[3].strip().split(" "))
+                players = l[3]
             client.notice(
-                self.channel,
-                "Pickup #{0}, {1} ago [{2}]: {3}".format(n, ago, gt, players),
+                self.channel, f"**Match {match_id} [{gametype}]:** {ago} ago\n{players}"
```

Comment:
This throws an exception:
```
(17:55:30)ERROR| Error processing message: Traceback (most recent call last):
  File "/home/alde/dev/Pubobot/pubobot/client.py", line 244, in on_message
    await channel.processmsg(message)
  File "/home/alde/dev/Pubobot/pubobot/bot.py", line 882, in processmsg
    self.lastgame(member, msgtup[1:msglen])
  File "/home/alde/dev/Pubobot/pubobot/bot.py", line 1288, in lastgame
    self.channel, f"**Match {match_id} [{gametype}]:** {ago} ago\n{players}"
                             ^^^^^^^^
NameError: name 'match_id' is not defined
```
It's hard to see because the bot swallows exceptions. I think `match_id` should be `pickup_num` in here. Or vice-versa.

If you run `poetry run pytest -vv -s` it will dump the entire bot interaction.